openfisca_web_site/templates/france/fr/tree/demo.mako

```diff
@@ -0,0 +1,245 @@
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"
+    integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+    crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+
+      //Variables globales
+      var salary1 = 25000;
+      var salary2 = 15000;
+
+      window.addEventListener("load", callOnLoad, false);
+
+      var updateSalaries = _.debounce(function(salary, montant) {
+          if (salary == "salary1") {
+            salary1 = Number(montant);
+          } else if (salary == "salary2") {
+            salary2 = Number(montant);
+          }
+          callOpenFiscaAPI();
+      }, 500);
+
+
+      function formatJSONOfSituation(){
+        return JSON.stringify({
+          "individus": {
+            "Bill": {
+              "salaire_de_base": {
+                "2017": salary1
+              },
+              "revenus_du_travail": {
+                "2017": null
+              }
+            },
+            "Bob": {
+              "salaire_de_base": {
+                "2017": salary2
+              },
+              "revenus_du_travail": {
+                "2017": null
+              }
+            }
+          },
+          "familles": {
+            "famille_1": {
+              "parents": [
+                "Bill",
+                "Bob"
+              ],
+              "prestations_sociales": {
+                "2017": null
+              }
+            }
+          },
+          "foyers_fiscaux": {
+            "foyer_fiscal_1": {
+              "declarants": [
+                "Bill",
+                "Bob"
+              ]
+            }
+          },
+          "menages": {
+            "menage_1": {
+              "conjoint": [
+                "Bob"
+              ],
+              "personne_de_reference": [
+                "Bill"
+              ],
+              "impots_directs": {
+                "2017": null
+              },
+              "revenu_disponible": {
+                "2017": null
+              }
+            }
+          }
+        });
+      }
+
+      function callOnLoad(){
```

Comment:
Function names should help us understand what they do at a glance. 

When I read `window.addEventListener("load", callOnLoad, false);`, I understand : when the window is loaded... call the function to call when the window is load. Which doesn't help 😆 

What about calling this function `initPage` ? or `preDisplayPage` ? This way the previous code becomes : 
`window.addEventListener("load", initPage, false);`. The intention is much clearer : when the window is loaded, initialise the page.